security/certificate_types_descriptions/aggregated-api-client-certificates.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="cert-types-aggregated-api-client-certificates"]
+= Aggregated API client certificates
+include::_attributes/common-attributes.adoc[]
+:context: cert-types-aggregated-api-client-certificates
+
+toc::[]
+
+== Purpose
+
+Aggregated API client certificates are used to secure connections between the KubeAPIServer and Aggregated API Servers.
```

Comment:
They are used to authenticate the KubeAPIServer when connecting to the Aggregated API Servers